### PR TITLE
Fix the wrong word and pronounce

### PR DESCRIPTION
**Pull request by D50000**
In DB structure we usually use word `schema` instead of `scheme`.

reference: 
- https://www.youtube.com/watch?v=9zvXH8Rzk7I
- https://www.youtube.com/watch?v=3BZz8R7mqu0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,7 +80,7 @@
 | [route](https://www.ldoceonline.com/dictionary/route) [🔊](https://www.ldoceonline.com/media/english/ameProns/route1.mp3) | ✅ [ruːt] |  |
 | [rust](https://www.ldoceonline.com/dictionary/rust)[🔊](https://www.ldoceonline.com/media/english/ameProns/rust1.mp3) | ✅ [rʌst] | ❌ [rust] |
 | [safari](https://www.ldoceonline.com/dictionary/safari) [🔊](https://www.ldoceonline.com/media/english/ameProns/safari1.mp3) | ✅ [sə'fɑːrɪ] | ❌ [sæfərɪ] |
-| [scheme](https://www.ldoceonline.com/dictionary/scheme) [🔊](https://www.ldoceonline.com/media/english/ameProns/scheme1.mp3) | ✅ [skiːm] | ❌ [s'kæmə] |
+| [schema](https://www.ldoceonline.com/dictionary/schema) [🔊](https://www.youtube.com/watch?v=9zvXH8Rzk7I) | ✅ [s'kæmə]  | ❌ [skiːm] |
```
Comment (pjchender) [inline README.md]: Should it pronounce as ``[`skimə]`` ?

**Comment by pjchender** [comment]
Please also update the pronunciation.

File: README.md (hunk 1 above)
Comment (D50000) [reply README.md]: Yes, schema. ( https://www.ldoceonline.com/dictionary/schema)
Scheme is wrong one.

More detail:
https://www.askdifference.com/scheme-vs-schema/

Comment (pjchender) [reply README.md]: I mean the schema should be pronounced as ``[`skimə]``

**Comment by D50000** [comment]
Done 👌👍

> Please also update the pronunciation.

**Comment by pjchender** [comment]
Thanks, will close the PR (#7).

**Pull request merged**